python/mxnet/ndarray/numpy/_op.py

```diff
@@ -293,3 +293,46 @@ def power(x1, x2, out=None):
         This is a scalar if both x1 and x2 are scalars.
     """
     return _ufunc_helper(x1, x2, _npi.power, _np.power, _npi.power_scalar, _npi.rpower_scalar, out)
+
+
+@set_module('mxnet.ndarray.numpy')
+def copysign(x1, x2, out=None):
```

Comment:
I think we will not support the `where` argument. May need to keep the incompatible behaviors in a note.

Reply:
Yes. Notes have been added in docs. See `_op.py`, `multiarray.py` and `_symbol.py`. Thank you!